keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,452 @@
+## Proposal
+
+This KEP proposes a transition to a *3-releases-per-calendar-year cadence*, beginning 
+with the Kubernetes 1.22 Release. This would result in a *15* week release 
+cycle, with *2* weeks between release cycles. During the Kubernetes 1.22 release, 
+a focused communication effort will be undertaken to communicate to contributors and 
+the end user community.
+
+The following tables detail a notional timeline for the remainder of 2021 and 
+for 2022, leveraging the historical *4-releases-per-calendar-year cadence*. Generally, 
+code freeze remains in effect until the last week of the release, so 
+development for the next release generally starts prior to the official release 
+team kickoff. A minimum of 1 week is needed between releases to fully form the 
+release team and to facilitate on-boarding of shadows. The fourth release of 
+the year has traditionally been compressed and limited in scope, overlapping 
+with end of year holidays and vacation for many contributors. Additionally, 
+KubeCon normally occurs during at least one release, eliminating at *least* one week of 
+working time.
+
+*Kubernetes Release Sechedule 2021 (Existing 4 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 2     | 1     | 1 (January 11) | |
+| 14 | 1 | 13 (April 8) | |
+| 16 | 2 | 1 (April 19) | |
+| 27 | 2 | 11 (July 06) | One week break for KubeCon EU | 
+| 29 | 3 | 1 (July 20) | |
+| 40 | 3 | 11 (October 5) | | 
+| 42 | 4 | 1 (October 18) | |
+| 52 | 4 | 10 (December 28) | End of Year Holidays |
+
+*Kubernetes Release Sechedule 2022 (Existing 4 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 3) | |
+| 12 | 1 | 12 (March 15) | |
+| 14 | 2 | 1 (March 28) | Probable KubeCon EU Break|
+| 26 | 2 | 12 (June 28) | |
+| 28 | 3 | 1 (July 11) | | 
+| 40 | 3 | 12 (October 4) | | 
+| 42 | 4 | 1 (October 17) | Probably KubeCon NA Break | 
+| 52 | 4 | 10 (Dec 28) | | 
+
+With the proposed change in cadence, the notional schedules for the remainder of
+2021 and 2022 are shown below:
+
+*Kubernetes Release Schedule 2021 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 2  | 1 | 1 (January 11) | |
+| 14 | 1 | 13 (April 8) | |
+| 17 | 2 | 1 (April 26) | |
+| 32 | 2 | 15 (August 02) | KubeCon EU Break (May 4-7) |
+| 35 | 3 | 1 (August 30) | | 
+| 50 | 3 | 15 (December 14) | Kubecon Break (Oct 12-15) | 
+
+*Kubernetes Release Sechedule 2022 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 3) | |
+| 15 | 1 | 15 (April 12) | | 
+| 18 | 2 | 1 (May 2) | Probably KubeCon EU |
+| 33 | 2 | 15 (August 15) | |
+| 36 | 3 | 1 (September 6 | Probably KubeCon NA | 
+| 51 | 3 | 15 (December 20) | 
```

Comment:
Er, you sure about a December 20th release date?  That seems very risky considering the proximity to the holidays. One hiccup and you're releasing in January.

Reply:
yeah, anything past December 10thish seems really iffy to me.

Reply:
Thanks for the input, I've updated this table to back that date up to the week of (Dec 06 - Dec 11)